Update general-remarks.md (#72)
MewsSystems/gitbook-channel-manager-api: general-remarks.md

| Property | Type |  | Description |
| --- | --- | --- | --- |
| `success` | `bool` | required | `true` or `false` determining result of operation. |

> | ~~`error`~~ | ~~[`Error`](general-remarks.md#error)~~ | ~~optional~~ | ~~In case of `"success": false`, this property holds information of the error that happened.~~ |

Reviewer: Do you have followup cleanup tasks for all such places?

Author: From the far past we dont now i will connect it with the deprecation issue.

Reviewer: The you should figure it out and remove all the things that are currently crossed out and have been like this for long time. You should also remove it from the API completely.

| `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
| `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
| `8` | Invalid authorization |
| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |

> | `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. | 

Reviewer 2: missing one word: space category XX availability updates cannot be accepted due to the settings in the channel manager extranet.

> | `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. | 

Author: ```suggestion
| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager extranet. | 
```

> | `11` | Rate category error - e.g. category was removed from the rate plan in channel manager extranet. |

Reviewer 2: e.g., such rate plan-space category combination is not created in the channel manager extranet.

| `12` | Availability configuration error - e.g., availability updates blocked from the channel manager extranet. |
| `13` | Prices configuration error - e.g., price updates blocked from the channel manager extranet. |

> | `14` | Restrictions configuration error - e.g. operation is disabled in channel manager extranet. |

Reviewer 2: Let's follow the same format everywhere, so it won't cause any confusion.

e.g., restriction updates blocked from the channel manager extranet.

> | `14` | Restrictions configuration error - e.g. operation is disabled in channel manager extranet. |

Author: ```suggestion
| `14` | Restrictions configuration error - e.g., restriction updates blocked from the channel manager extranet. |
```
